openlcsd/flow/tasks.py

```diff
@@ -78,6 +79,35 @@ def get_config(context, engine):
     engine.logger.info("Worker {}".format(socket.gethostname()))
 
 
+def filter_duplicated_import(context, engine):
+    url = 'components/filter'
+    cli = context.get('client')
+    component_type = context.get('component_type')
+    build_data = context.get('build', {})
+    data = {
+        'name': build_data.get('name', ''),
+        'version': build_data.get('version', ''),
+        'release': build_data.get('release', ''),
+        'type': component_type,
+        'license_scan': context.get('license_scan'),
+        'copyright_scan': context.get('copyright_scan')
+    }
+    if component_type == 'RPMMOD':
+        data['arch'] = ''
+    else:
+        data['arch'] = build_data.get('arch', 'src')
+    msg = 'Start to filter duplicated import for {}'.format(data)
+    engine.logger.info(msg)
+    resp = cli.post(url, data=data)
+    if resp.status_code == 200:
+        should_filter = resp.json()['filter_results']['should_filter']
+        if should_filter:
+            task_id = context.get('task_id')
+            res = AsyncResult(task_id)
+            res.revoke(terminate=True)
```

Comment:
There should be a prompt message about why the task is terminated.

Reply:
DONE